ui/admin/app/components/scope-options/index.js

```diff
@@ -0,0 +1,144 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import {
+  GRANT_SCOPE_THIS,
+  GRANT_SCOPE_CHILDREN,
+  GRANT_SCOPE_DESCENDANTS,
+  GRANT_SCOPE_KEYWORDS,
+} from 'api/models/role';
+
+export default class ScopeOptionsIndexComponent extends Component {
+  // =attributes
+
+  keywords = {
+    keyThis: GRANT_SCOPE_THIS,
+    keyChildren: GRANT_SCOPE_CHILDREN,
+    keyDescendants: GRANT_SCOPE_DESCENDANTS,
+  };
+
+  /**
+   * Returns root scope displayName.
+   * @type {string}
+   */
+  get scopeDisplayName() {
+    if (this.args.model.scope.isGlobal) {
+      return 'Global';
+    }
+    return this.args.model.scope.name ?? this.args.model.scope.id;
```

Comment:
We should be able to use the `displayName` getter here.

Reply:
oh yup good catch!